extensions/wikia/AdEngine/js/recovery/gcs.js

```diff
@@ -0,0 +1,86 @@
+/*global define*/
+/*jshint camelcase:false*/
+define('ext.wikia.adEngine.recovery.gcs', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adTracker',
+	'ext.wikia.adEngine.recovery.helper',
+	'jquery',
+	'wikia.document',
+	'wikia.location',
+	'wikia.log',
+	'wikia.scriptwriter',
+	'wikia.window'
+], function (
+	adContext,
+	adTracker,
+	recoveryHelper,
+	$,
+	doc,
+	loc,
+	log,
+	scriptWriter,
+	win
+) {
+	'use strict';
+	var articleUrl = loc.href,
+		contentId = 'everything',
+		context = adContext.getContext(),
+		logGroup = 'ext.wikia.adEngine.recovery.gcs';
+
+	function addClasses() {
+		var article = $('#WikiaArticle');
+		article.addClass('p402_premium');
+		article.find('table,figure,.portable-infobox').each(function () {
+			$(this).addClass('p402_hide');
+		});
+	}
+
+	function trackPosition() {
+		var survey = $('.t402-prompt-iframe-container'),
```

Comment:
`document.getElementsByClassName('.t402-prompt-iframe-container')` is the [vanilla-js](http://vanilla-js.com/) way to do it :japanese_ogre: